src/pybullet_industrial/interpolation.py

```diff
@@ -118,34 +138,73 @@ def circular_interpolation(start_point: np.array, end_point: np.array,
     planar_path = planar_circular_interpolation(
         planar_start_point, planar_end_point, radius, samples, clockwise)
 
-    path = np.zeros((3, samples))
+    positions = np.zeros((3, samples))
     for i in range(2):
-        path[all_axis[i]] = planar_path[i]
-    path[axis] = np.linspace(start_point[axis], end_point[axis], samples)
-    return ToolPath(path)
+        positions[all_axis[i]] = planar_path[i]
+    positions[axis] = np.linspace(start_point[axis], end_point[axis], samples)
+    final_path = ToolPath(positions=positions)
+
+    if start_orientation is not None and end_orientation is not None:
+        final_path.orientations = orientation_interpolation(
+            start_orientation, end_orientation, samples=samples)
 
+    return final_path
 
-def spline_interpolation(points: np.array, samples: int):
+
+def orientation_interpolation(start_orientation: np.array,
+                              end_orientation: np.array, samples: int):
+    """Interpolates between two orientations
+
+    Args:
+        start_orientation (np.array(3,1)): 3-D array containing the start orientation
+        end_orientation (np.array(3,1)): 3-D array containing the end orientation
+        samples (int): The number of samples used to interpolate
+    """
+    start_orientation = p.getEulerFromQuaternion(start_orientation)
+    end_orientation = p.getEulerFromQuaternion(end_orientation)
+    euler_orientations = np.linspace(start_orientation,

```

Comment:
What happens if the Euler angles cross over from -2pi to 0. It should be checked that this behaves as expected